docs/introduction/entity-component-system.md

```diff
@@ -451,27 +451,37 @@ Now we can include it into our HTML:
 </html>
 ```
 
+### Using jsdelivr for CDN
+
+[jsdelivr]: https://www.jsdelivr.com/
+
+[JSDELIVR][jsdelivr] is an alternative CDN to unpkg.  One benefit of JSDELIVR is that it can download files from GitHub as well as NPM.
+
+You can convert unpkg URLs to  JSDELIVR URLs using this link: https://www.jsdelivr.com/unpkg
+
+You can convert GitHub URLs to  JSDELIVR URLs using this link: https://www.jsdelivr.com/github
+
 ### Example
 
-[glitch]: http://glitch.com/~aframe-registry
+[community-example]: https://aframe.io/examples/showcase/community-components
 
-![Registry Example](https://cloud.githubusercontent.com/assets/674727/25502318/0f76ceec-2b4b-11e7-9829-cb3784b20dc1.gif)
+![Community Components Example](https://cloud.githubusercontent.com/assets/674727/25502318/0f76ceec-2b4b-11e7-9829-cb3784b20dc1.gif)
 
 Below is a complete example of using various community components from the
-Registry and using the unpkg CDN. We can [remix or check out this example on
-Glitch][glitch].
+Registry and using the JSDELIVR CDN.  This example can also be viewed in the [A-Frame Examples][community-example].
 
 ```html
 <html>
   <head>
-    <script src="https://aframe.io/releases/1.4.0/aframe.min.js"></script>
-    <script src="https://unpkg.com/aframe-animation-component@3.2.1/dist/aframe-animation-component.min.js"></script>
-    <script src="https://unpkg.com/aframe-particle-system-component@1.0.x/dist/aframe-particle-system-component.min.js"></script>
-    <script src="https://unpkg.com/aframe-extras.ocean@%5E3.5.x/dist/aframe-extras.ocean.min.js"></script>
-    <script src="https://unpkg.com/aframe-gradient-sky@1.4.0/dist/gradientsky.min.js"></script>
+    <title>Community Components Example</title>
```

Comment:
We now have duplicated code here and the example that have to be in sync. It would be cool if there's a markdown way to display html from the file hosted on gihub

Reply:
One option:
https://github.com/zakhenry/embedme

A limitation is that code fragments are indexable only by line number.  If line numbers change, extracts will get out of sync.  That's discussed (together with some possible solutions) [here](https://github.com/zakhenry/embedme/issues/48) but there doesn't seem to be a good solution yet (and momentum has stalled since 2020).

How widespread an issue do you think this is in the docs?  Is there enough benefit to be worth the overhead of adding new tools like embedme to the pipeline?  Is it worth doing if snippets have to be by line number, rather than tag-based as #48 will enable?